classes/game.rb

```diff
@@ -0,0 +1,18 @@
+require 'date'
+require_relative './item'
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_date, :publish_date
+
+  def initialize(multiplayer, last_played_date, publish_date)
+    super(name, publish_date)
+    @last_played_date = last_played_date
+    @multiplayer = multiplayer
+  end
+
+  def can_be_archived?
+    current_year = Time.new.year
+    publish_year = Date.strptime(@publish_date, '%Y-%m-%d').strftime('%Y').to_i
+    current_year - publish_year >= 10
+  end
```

Comment:
- [ ] This method returns true when the publishing date and the present date differ or equal 10, BUT it should return true ONLY when **`@last_played_date` is greater than 2**

  ![ranjeet-game](https://user-images.githubusercontent.com/36542986/158573665-26d7e986-dd4e-49d0-9b20-70a81a991f4b.jpg)